scripts/src/main/resources/scripts/command/ide

```diff
@@ -318,8 +318,9 @@ function doUpdateSoftware() {
   local tool
   for tool in "${DEVON_IDE_TOOLS[@]}"
   do
-    doEcho "\n*** Setting up ${tool} ***"
+    doEcho -e "\e[96m\n*** Setting up ${tool} ***\e[39m"
```

Comment:
Did you do search&replace of `echo` case insensitive?
`-e` is wrong here for `doEcho`.
See here: 
https://github.com/devonfw/ide/blob/d462bc6ae93075f613081d710cd88f1490fd1ed8/scripts/src/main/resources/scripts/functions#L149

Reply:
Please note there is also `doWarning` and `doDebug`, etc.
We could add the color automatically for `doWarning` what would make much more sense then hardcoding this strange escape syntax redundantly everywhere...